perma_web/frontend/lib/helpers.js

```diff
@@ -6,4 +6,8 @@ export function getCookie(name) {
       "(?:^|; )" + name.replace(/([\.$?*|{}\(\)\[\]\\\/\+^])/g, '\\$1') + "=([^;]*)"
   ));
   return matches ? decodeURIComponent(matches[1]) : undefined;
-}
\ No newline at end of file
+}
+
+export function prefersReducedMotion() {
+  return window.matchMedia('(prefers-reduced-motion: reduce)').matches;
+}
```

Comment:
Can you say a word about how this differs from the `import { usePreferredReducedMotion } from '@vueuse/core'` strategy (other than being more readable 😜). Is it the same thing, under the hood?

Reply:
It's the same media query -- I just found `usePreferredReducedMotion().value === 'reduced'` clunky enough that I wanted a helper for it, and the helper simple enough not to require a library.

usePreferredReducedMotion() does [a bunch of work](https://github.com/vueuse/vueuse/blob/main/packages/core/usePreferredReducedMotion/index.ts#L13) to be reactive, so if we wanted to watch and respond to changes in reduced motion it would be helpful.